<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StatusBarStat</class>
 <widget class="QWidget" name="StatusBarStat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>24</width>
    <height>16</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>1</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>7</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="caption">
     <property name="indent">
      <number>9</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="value">
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="max">
     <property name="visible">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
  <action name="action_display">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Display</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="action_show_max">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Max</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="action_set_color">
   <property name="text">
    <string>Set Color...</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="action_set_caption_color">
   <property name="text">
    <string>Set Label Color...</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="action_set_value_color">
   <property name="text">
    <string>Set Value Color...</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="action_reset_colors">
   <property name="text">
    <string>Reset Colors</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>action_display</sender>
   <signal>toggled(bool)</signal>
   <receiver>StatusBarStat</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>11</x>
     <y>7</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_show_max</sender>
   <signal>toggled(bool)</signal>
   <receiver>max</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>49</x>
     <y>14</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
